pulp_container/app/viewsets.py

```diff
@@ -545,6 +546,29 @@ class ContainerRepositoryVersionViewSet(RepositoryVersionViewSet):
     """
 
     parent_viewset = ContainerRepositoryViewSet
+    permission_classes = (AccessPolicyFromDB,)
+
+    DEFAULT_ACCESS_POLICY = {
+        "statements": [
+            {
+                "action": ["list"],
+                "principal": "authenticated",
+                "effect": "allow",
+            },
+            {
+                "action": ["retrieve"],
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": "has_repo_attr_model_or_obj_perms:container.view_containerrepository",
+            },
+            {
+                "action": ["destroy"],
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": "has_model_or_obj_perms:delete_containerrepository_versions",
```

Comment:
apologies if this is not ready for review yet, i have briefly skimmed and noticed we miss an OR here for delete_container_repo  permission